src/com/xilinx/rapidwright/rwroute/RWRoute.java

```diff
@@ -289,14 +299,79 @@ private void determineRoutingTargets(){
 				System.err.println("ERROR: Unknown net " + net.toString());
 			}
 		}
-		if(this.config.isVerbose()) this.printConnectionSpanStatistics();
+	}
+	
+	/**
+	 * Deals with the conflicted nets.
+	 * Note: this is customized for the RapidStream use case. Other users will need to adapt this method.
+	 * @param design The design to route.
+	 * @param conflictNets A set of conflicted nets.
+	 */
+	private void handleConflictNets(Design design, Set<Net> conflictNets) {
+		List<Net> toPreserveNets = new ArrayList<>();
+		for(Net net : conflictNets) {
+			if(net.getType() != NetType.WIRE) {
+				// Skip successfully routed CLK, VCC, and GND nets
+				toPreserveNets.add(net);
+				continue;
+			}
+			
+			if(net.getSinkPins().size() > 1) {
+				// In the RapidStream flow, the target nets to route are 2-terminal FF-to-FF nets.
+				toPreserveNets.add(net);
+				continue;
+			}
+			
+			if(!this.isRegularAnchorNet(net, design)) {
+				toPreserveNets.add(net);
+				continue;
+			}
```

Comment:
```suggestion
			if(net.getType() != NetType.WIRE || net.getSinkPins().size() > 1 || !this.isRegularAnchorNet(net, design)) {
				// Skip successfully routed CLK, VCC, and GND nets
				// In the RapidStream flow, the target nets to route are 2-terminal FF-to-FF nets.
				toPreserveNets.add(net);
				continue;
			}
```

The bodies of the three `if` statements seem redundant.